interface-definitions/service_dns_forwarding.xml.in

```diff
@@ -793,6 +793,155 @@
                   </leafNode>
                 </children>
               </node>
+              <tagNode name="zone-to-cache">
+                <properties>
+                  <help>Load a zone into the recursor cache</help>
+                  <valueHelp>
+                    <format>txt</format>
+                    <description>The name of the zone to load</description>
+                  </valueHelp>
+                </properties>
+                <children>
+                  <node name="source">
+                    <properties>
+                      <help>Source to load zone</help>
+                    </properties>
+                    <children>
+                      <leafNode name="axfr">
+                        <properties>
+                          <help>Source IP address</help>
+                          <valueHelp>
+                            <format>ipv4</format>
+                            <description>IPv4 address</description>
+                          </valueHelp>
+                          <constraint>
+                            <validator name="ipv4-address"/>
+                          </constraint>
+                        </properties>
+                      </leafNode>
+                      <leafNode name="url">
+                        <properties>
+                          <help>Source URL</help>
+                          <valueHelp>
+                            <format>url</format>
+                            <description>Set URL location</description>
```

Comment:
No need for a verb in this help string. Just "Zone file URL".

Reply:
done